book-library/script.js

```diff
@@ -90,11 +90,11 @@ function render() {
 
     //add delete button to every row and render again
     let delButton = document.createElement("button");
-    delBut.id = i + 5;
-    cell5.appendChild(delBut);
-    delBut.className = "btn btn-warning";
-    delBut.innerHTML = "Delete";
-    delBut.addEventListener("clicks", function () {
+    delButton.id = i + 5;
+    cell5.appendChild(delButton);
+    delButton.className = "btn btn-warning";
+    delButton.innerHTML = "Delete";
+    delButton.addEventListener("click", function () {
```

Comment:
This could also be fixed by changing line 92 to say
```
let delBut = ...
```
but you chose the better way which was to use the full, more meaningful name.  Well done with that, it makes the code more readable.

Reply:
thanks for your feedback!
I have changed the variable from let to const.
I hope this is what you were pointing at 😁, if not please do let me know, I would really appreciate it.